scout/templates/scout/view_classes.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %}
+{% block title %}Enrolled Courses{% endblock title %}
+{% block header_text %}Enrolled Courses{% endblock header_text %}
+
+{% block content %}
+
+<table class="table">
+	<thead>
+        <tr>
+            <td>Course Name</td>
```

Comment:
I'm wondering if we should make an object specifically for this view that can render itself?